report/report-generator.js

```diff
@@ -102,13 +93,58 @@ class ReportGenerator {
     return '<script src="/pages/scripts/lighthouse-report.js"></script>';
   }
 
+  _createSections(aggregations) {
+    return aggregations.reduce((menu, aggregation) => {
+      if (menu.indexOf(aggregation.type.name) === -1) {
+        menu.push(aggregation.type.name);
+      }
+      return menu;
+    }, []);
+  }
+
+  _createPWAAuditsByTag(aggregations) {
```

Comment:
maybe a comment for what this is for, pulling audits from across aggregations to create a special PWA section? (some more comments in this file would be good in general)